politeiad/cache/cockroachdb/decred.go

```diff
@@ -203,6 +207,11 @@ func (d *decred) voteResultsMissing(bestBlock uint64) ([]string, []string, error
 		runoff = append(runoff, token)
 	}
 
+	// Update latest best block used to run this procedure
+	d.bbMtx.Lock()
+	d.lastBestBlock = bestBlock
+	d.bbMtx.Unlock()
```

Comment:
Have you gone through all the decred plugin commands and made sure you're updating the best block anytime its passed in as a param?

Reply:
Was missing an update on `voteSummaries`. Done.